ssl/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply from: "$rootDir/gradle/javaModule.gradle"
+
+archivesBaseName = 'ssl'
+group = 'io.crate'
+description = 'SSL Encryption for CrateDB'
+
+dependencies {
+    compile project(':core')
```

Comment:
I included this for logging. Can also replace this with just the log4j dependency.